robolectric-shadows/shadows-core/src/main/java/org/robolectric/fakes/RoboMenu.java

```diff
@@ -52,7 +51,7 @@ public MenuItem add(int groupId, int itemId, int order, CharSequence title) {
 
   @Override
   public MenuItem add(int groupId, int itemId, int order, int titleRes) {
-    return add(groupId, itemId, order, context.getResources().getString(titleRes));
+    return add(groupId, itemId, order, RuntimeEnvironment.application.getResources().getString(titleRes));
```

Comment:
Just curious but why didn't you stick with just using the context passed in the constructor, I think this is the better approach since the resources you get may be context dependent.